vendor.mod

```diff
@@ -120,9 +104,3 @@ replace (
 // After updating GRPC, if integration test failures occur, verify that the
 // string matching there is correct.
 replace google.golang.org/grpc => google.golang.org/grpc v1.23.0
```

Comment:
Do we need to / should we copy the replace rules from the containerd version we're using? Or does it break with v1.27.1 ? There's some other related replace in the containerd go.mod as well; https://github.com/containerd/containerd/blob/v1.5.9/go.mod#L75-L80


```
	github.com/gogo/googleapis => github.com/gogo/googleapis v1.3.2
	github.com/golang/protobuf => github.com/golang/protobuf v1.3.5
	// urfave/cli must be <= v1.22.1 due to a regression: https://github.com/urfave/cli/issues/1092
	github.com/urfave/cli => github.com/urfave/cli v1.22.1
	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20200224152610-e50cd9704f63
	google.golang.org/grpc => google.golang.org/grpc v1.27.1
```

Reply:
This is another non-trivial one, so I'd leave it for another PR :)

Reply:
Yeah this is the hard one that blocks buildkit vendoring on moby.

Reply:
Yeah, we can do in a follow-up. I was looking at the moby code, which already included those replace rules (linking to the vendor.conf file for convenience) and versions of containerd/runc

- https://github.com/moby/moby/blob/79651b7a979b40e26af353ad283ca7ea5d67a855/vendor.conf#L83
- https://github.com/moby/moby/blob/79651b7a979b40e26af353ad283ca7ea5d67a855/vendor.conf#L76

For these I was more concerned about not having replace rules for some of them, but CI is green so at least currently it works 😂